fw-child/functions.php

```diff
@@ -539,6 +540,25 @@ function cdc_enable_block_editor( $use_block_editor, $post_type ) {
 	return false;
 }
 
+/**
+ * Returns a an array of args making interactive posts public if loaded by Motion.page editor.
```

Comment:
The description may be a little bit hard to understand. A suggestion:

```suggestion
 * Updates the `interactive` post type arguments to make it public if loaded by the motion.page editor.
```